Entity/EntityTrait/DateTimeControlTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Grossum\CoreBundle\Entity\EntityTrait;
+
+/**
+ * @deprecated since version 2.0
```

Comment:
`@deprecated since 0.1.1 and will be removed in 0.2. Use DatesAwareTrait instead`